aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -777,7 +776,7 @@ def filter_tables(self, db: PGDatabase) -> List[str]:
                     ):
                         found = t
                         break
-                if not found:
+                if not found and t not in ret:
```

Comment:
This `and t not in ret` has no effect. Same below. Doesn't really hurt either but I guess we could make do without this change.